### ISSUE TITLE
restoreAllowFlight not working

### ISSUE THREAD

**Issue by bosheltuts** [AuthMe/AuthMeReloaded#1582]
it doesn't restore the flying state properly, I'm using the default config.
if I logout in the air while flying, it will disable fly at next login also with NOTHING still not work

**Comment by ljacqu** [comment]
How can we reproduce? What version (of AuthMe and Minecraft server) are you using?

**Comment by bosheltuts** [comment]
- CraftBukkit version git-Spigot-2086bb0-8cc5a7e (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
- AuthMeReloaded v5.4.0 b1877

It doesn't restore the flying state properly that means when i logoud in the air while flying, it disable fly at the next login so the players will get fall damage even if I had the essentials.fly.safelogin permission.
I've tried with RESTORE and with NOTHING in the restoreAllowFlight config. :(

**Comment by ljacqu** [comment]
Hmm, if NOTHING still has that same behavior then I‘d wager that it‘s another plugin that changes the state — Essentials maybe?
One quick test would be whether the behavior is still the same if you remove AuthMe. I bet it is. 
I think we‘ve had this reported in the past; I‘ll try to find the issue

**Comment by bosheltuts** [comment]
Yaap If I remove AuthMe plugin it works fine, I login with my flying state properly, hope you can find it, and thanks for the prompt response.

**Comment by sgdc3** [comment]
@bosheltuts does it happen with only authme enabled?

**Comment by bosheltuts** [comment]
Yes, when I uninstall Authme, essentials does its job fine, it restore the flight mode in de position I logout, also Take the opportunity to tell you in
 settings:
      customJoinMessage: ''
I can't add custom colors to the custom join message, the message when a player join the server it only appears in white color

**Comment by bosheltuts** [comment]
Yes, when I uninstall Authme, essentials does its job fine, it restore the flight mode in the position I logout, also Take the opportunity to tell you in
_settings:
customJoinMessage: ''_
I can't add custom colors to the custom join message, the message when a player join the server it only appears in white color

**Comment by r4nx** [comment]
Have this bug too. AuthMe 5.4.0-b1877 and Essentials 2.15.0.16. To reproduce just /logout while flying.

**Comment by niqoar** [comment]
Same bug with:
AuthMeReloaded version v5.6.0-SNAPSHOT (build: 2314)
EssentialsX version: 2.17.2.0
Server version: 1.15.2-R0.1-SNAPSHOT git-Paper-118 (MC: 1.15.2)

**Comment by Cigneture** [comment]
This issue still persists in AuthMeReloaded 5.6.0-Beta2 on PaperSpigot 1.17.1.

**Comment by PaBox** [comment]
I can actually not support that statement. I use Java 16, lots of other plugins, including essentials and all multiverse plugins on 1.16.5. I set the option in AuthMeReloaded to NOTHING and it does work for me :) So this might not be a generic problem.

**Comment by Cigneture** [comment]
You might have another plugin installed that allows you to remain in flight.

Here's a video I recorded, I connect to the server twice, once with the plugin enabled and once with the plugin disabled.

https://streamable.com/v8y0au

**Comment by Cigneture** [comment]
You might have another plugin installed that allows you to remain in flight.

Here's a video I recorded, I connect to the server three times. Once to confirm that fly was indeed enabled before disconnecting, a second time to confirm that flight is revoked upon login and a third login with the plugin disabled to confirm that flight was not being revoked by another plugin (e.g. EssentialsX, Multiverse etc.).

https://streamable.com/apmb26

**Comment by Cigneture** [comment]
You might have another plugin installed that allows you to remain in flight.

Here's a video I just recorded. I connected to the server three times. Once to confirm that /fly was indeed enabled before disconnecting, a second time to confirm that flight was being revoked upon login and a third login with AuthMeReloaded disabled to confirm that flight was not being revoked by another plugin (e.g. EssentialsX, Multiverse etc.), restoreAllowFlight was set to NOTHING.

https://streamable.com/apmb26

**Comment by krusic22** [comment]
We don't interact with EssentialsX for flight. So, no. AuthMe will not restore EssentialsX /fly.
